samples/skills/QASkill/GitHubMemoryQuery/skprompt.txt

```diff
@@ -0,0 +1,6 @@
+{{textmemoryskill.recall $input}}
```

Comment:
I'm still unclear if it makes sense for this to be part of our sample skills or if it belongs in the `github-skills` project with the other github specific functions. Thoughts?

Reply:
I thought about moving it over to the github-skills project, then I saw that it was under this path: 
<img width="186" alt="image" src="https://user-images.githubusercontent.com/146438/226405478-ca32e815-1e53-4f69-89a6-50dd4072cfcd.png">

If we were to move it over there, it would make more sense to make it a native skill, which seems unnecessary?

Reply:
Fair point, let's clean this up in a subsequent PR if that's ok